snappy_pipeline/workflows/helper_gcnv_model_wgs/__init__.py

```diff
@@ -196,6 +196,7 @@ def get_resource_usage(self, action: str, **kwargs) -> ResourceUsage:
 
         def get_memory(wildcards, input=None, threads=None, attempt=None):
             _, _, _ = wildcards, input, threads  # unused but cannot be renamed
+            attempt = attempt or 0
```

Comment:
Hm, did this ever work before? Shouldn't attempt default to 1?

Reply:
It must have worked somehow (and I'm a bit afraid the attempt var is now not making it through anymore, since this only failed in "live testing").
If snakemake defaults to 1 we should also

Reply:
Does snakemake only give/use named parameters for input functions?